integration/test_collection_config.py

```diff
@@ -3,6 +3,8 @@
 import weaviate
 from weaviate import Config
 from weaviate.collection.classes.config import (
+    _CollectionConfig,
+    _CollectionConfigSimple,
```

Comment:
hmm what is the point of having two different ones?

Reply:
@databyjp, requested the ability to return a simplified object that only has the basics present